src/polyfills/datepicker/datepicker.js

```diff
@@ -151,7 +151,7 @@ var componentName = "wb-date",
 		// Disable the tabbing of all the links when calendar is hidden
 		$container.find( "a" ).attr( "tabindex", "-1" );
 
-		$( "main" ).after( $container );
+		$( elm ).after( $container );
```

Comment:
This was the way it was before and it was switched to be in the overlay to avoid a bunch of issues, so merging this would bring back these issues. Also, this way, there is only one instance of the datepicker that gets moved to the field once needed,. Applying this would mean that only the last field of the page woudl work and not the others ones